app/Http/Controllers/Auth/ResetPasswordController.php

```diff
@@ -25,7 +25,7 @@ class ResetPasswordController extends Controller
      *
      * @var string
      */
-    protected $redirectTo = '/home';
+    protected $redirectTo = '/reading-items';
```

Comment:
After reset password we log them in and take them to /reading-items or do we ask them to login ?

Reply:
Laravel logs in the user after a password reset. So it takes them to /reading-items

https://laravel.com/docs/5.6/passwords#after-resetting-passwords